doc/man1/flux-job.rst

```diff
@@ -119,14 +119,15 @@ results are consumed.
 When run with a jobid argument, ``flux job wait`` blocks until the specified
 job completes.  If the job was successful, it silently exits with a code of
 zero.  If the job has failed, an error is printed on stderr, and it exits with
-a code of one.  It is an error if the job was not submitted with the
-``waitable`` flag.
+a code of one.  If the jobid is invalid or the job is not waitable, ``flux job wait``
+exits with two.  This special exit code of two is used to differentiate between
```

Comment:
To be consistent, should be "a code of two".   I agree that's starting to get a little long winded but consistentcy...